src/SportsHub.Infrastructure/Repositories/CategoryRepository.cs

```diff
@@ -31,6 +31,30 @@ public async Task AddCategoryAsync(Category category)
             await _context.SaveChangesAsync();
         }
 
+        public async Task<Guid> GetCategoryIdByNameAsync(string categoryName)
+        {
+            var categories = await _context.Categories.ToListAsync();
+            var foundCategory = categories.Find(category => category.Name == categoryName);
+
+            if (foundCategory == null)
+            {
+                return Guid.Empty;
+            }
+            else return foundCategory.Id;
```

Comment:
```suggestion
            var category = await _context.Categories.Find(category => category.Name == categoryName);

            if (category == null)
            {
                return Guid.Empty;
            }
            else
            {
                return foundCategory.Id;
            }
```

Also, consider creating simply `GetByName()` instead

Reply:
great idea, thx